server/opts.go

```diff
@@ -439,6 +439,9 @@ type MQTTOpts struct {
 	// replicas count (lower than StreamReplicas if specified, but also lower
 	// than the automatic value determined by cluster size).
 	// Note that existing consumers are not modified.
+	//
+	// UPDATE: This is no longer used while messages stream has interest policy retention
```

Comment:
Should we mark as `Deprecated`?
```suggestion
	// Deprecated: This is no longer used while messages stream has interest policy retention
```

Reply:
I decided against in just in case I change over the stream to not be interest policy based in 2.10.